docs/examples/use_cases/index.rst

```diff
@@ -0,0 +1,9 @@
+Use Cases
+=========
+
+.. toctree::
+   :maxdepth: 2
+
+   detection_pipeline.ipynb
```

Comment:
This is a vision example.

Reply:
This detection pipeline is basically what we use in SSD, so I figured it's more of a use-case. Even disregarding the SSD, this is basically a use-case, as there's more to it than only image processing (e.g. labels passing and stuff).

Reply:
Following that logic any complete pipeline is a use case - RN50, SSD, SuperRes...
I still think that `Use Cases` is a vague idea that could only confuse people.